jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -115,30 +125,6 @@ export const usePipeline = (
     };
   }
 
-  const callSavePipeline = async () => {
-    setErrorMessage('');
-
-    let inputSpecJson;
-    try {
-      inputSpecJson = parseInputSpec(inputSpec);
```

Comment:
I believe you changes make `parseInputSpec` unneeded so you can delete. (GH won't let me comment on that section of code)

Reply:
Done! Good catch.